test/sort.spec.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const _ = require('lodash')
+const Promise = require('bluebird')
+const Joi = require('joi')
+const Hapi = require('hapi')
+
+let server, buildServer, destroyServer, hh;
+
+const schema = {
+    type: 'brands',
+    attributes: {
+        code: Joi.string().min(2).max(10),
+        description: Joi.string(),
+        year: Joi.number()
+    }
+};
+
+const data = {
+    attributes: {
+        code: 'MF',
+        description: 'Massey Furgeson',
+        year: 2000
+    }
+};
+
+
+//TODO just done the validation, actual sorts is remaining
```

Comment:
TODO should probably be removed as basic sort seems to be implemented : https://github.com/agco/hapi-harvester/pull/1/files#diff-55a387780cc6221ae09789e1dc0f4517R34

The missing feature is support for a descending sort